### DISCUSSION TITLE
[Question][Hw0] dune test execution

### DISCUSSION THREAD

**Discussion by john-mai-2605** [prosyslab-classroom/cs348-information-security#484] [Q&A]
When I use dune exec or dune test, this error occurred:

MoTTY X11 proxy: Unsupported authorisation protocol
can't open display: localhost:10.0

I supposed this is due to Mobaxterm (I use Mobaxterm to access the remote server). May anyone help me on how to solve this?
Thank you.

**Comment by mickowale** [comment]
I don't know about your problem but you may want to try out VSCode if you haven't already. You can download one of the Remote-SSH extensions to connect to the server.

**Comment by john-mai-2605** [comment]
Since VS Code solves the problem, I will close the issue
